Dockerfile

```diff
@@ -0,0 +1,3 @@
+ARG ARCH=
```

Comment:
What's reason for this change? Is this related to this PR if not, let's create separate PR